runtime/src/app/types.ts

```diff
@@ -4,9 +4,15 @@ export interface HydratedTarget {
 	redirect?: Redirect;
 	preload_error?: any;
 	props: any;
-	branch: Array<{ Component: DOMComponentConstructor, preload: (page) => Promise<any>, segment: string }>;
+	branch: Branch;
 }
 
+export type Branch = Array<{
```

Comment:
this type is a bit different from the one defined in https://github.com/sveltejs/sapper/pull/1528/files#diff-d1d1a1a1a8896e90a29cc23a07d69ed6R10

Reply:
i updated it with the types from the typings PR now.